elastic/kibana
# [Security Solution][Detection Engine] adds EBT telemetry for rule preview

## Summary

 - adds basic EBT telemetry for rule preview

### To test 

Use Discover Data View in staging to see reported events: https://telemetry-v2-staging.elastic.dev/s/securitysolution/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:60000),time:(from:now-28h,to:now))&_a=(columns:!(properties.ruleType,properties.loggedRequestsEnabled),filters:!(),index:security-solution-ebt-kibana-browser,interval:auto,query:(language:kuery,query:'event_type%20:%20%22Preview%20rule%22'),sort:!(!(timestamp,desc)))

Note, there is a few hours delay from event reported locally to be stored on staging host
